tutorials/stats-sensor-space/75_cluster_ftest_spatiotemporal.py

```diff
@@ -97,28 +109,58 @@
 # Then we generate a distribution from the data by shuffling our conditions
 # between our samples and recomputing our clusters and the test statistics.
 # We test for the significance of a given cluster by computing the probability
-# of observing a cluster of that size. For more background read:
-# Maris/Oostenveld (2007), "Nonparametric statistical testing of EEG- and
-# MEG-data" Journal of Neuroscience Methods, Vol. 164, No. 1., pp. 177-190.
-# doi:10.1016/j.jneumeth.2007.03.024
-
-
-# set cluster threshold
-threshold = 50.0  # very high, but the test is quite sensitive on this data
-# set family-wise p-value
-p_accept = 0.01
-
+# of observing a cluster of that size :footcite:`MarisOostenveld2007`
+# :footcite:`Sassenhagen2019`.
```

Comment:
FYI you can combine `footcites` like this:

```
:footcite:`MarisOostenveld2007,Sassenhagen2019`
```

Reply:
done (with some extra) in 3a569eb538a7f768123a10e2a6d349a0e5fa0b59